dm/dm/config/task.go

```diff
@@ -324,19 +324,19 @@ type SyncerConfig struct {
 	AutoFixGTID bool `yaml:"auto-fix-gtid" toml:"auto-fix-gtid" json:"auto-fix-gtid"`
 	EnableGTID  bool `yaml:"enable-gtid" toml:"enable-gtid" json:"enable-gtid"`
 	// deprecated
-	DisableCausality bool `yaml:"disable-detect" toml:"disable-detect" json:"disable-detect"`
-	SafeMode         bool `yaml:"safe-mode" toml:"safe-mode" json:"safe-mode"`
+	DisableCausality       bool   `yaml:"disable-detect" toml:"disable-detect" json:"disable-detect"`
+	SafeMode               bool   `yaml:"safe-mode" toml:"safe-mode" json:"safe-mode"`
+	SafeModeResumeDuration string `yaml:"safe-mode-resume-duration" toml:"safe-mode-resume-duration" json:"safe-mode-resume-duration"`
 	// deprecated, use `ansi-quotes` in top level config instead
 	EnableANSIQuotes bool `yaml:"enable-ansi-quotes" toml:"enable-ansi-quotes" json:"enable-ansi-quotes"`
 }
 
 // DefaultSyncerConfig return default syncer config for task.
 func DefaultSyncerConfig() SyncerConfig {
 	return SyncerConfig{
-		WorkerCount:             defaultWorkerCount,
-		Batch:                   defaultBatch,
-		QueueSize:               defaultQueueSize,
-		CheckpointFlushInterval: defaultCheckpointFlushInterval,
```

Comment:
I see it's adjusted again in SubTaskConfig.Adjust. Maybe we should also add an adjust in TaskConfig.adjust